### PR TITLE
Draft of super simple publish working for SSX

### PR DESCRIPTION
**Pull request by NickolausDS**
This properly plots and publishes now. I also removed the globus-automation code, and greatly simplified the funcx-functions. The pilot step is still required below:

Pilot install: 
```
pip install globus-pilot
pilot index set 5e63bb08-5b39-4a02-86f3-44cec03e8bc0
```

All tools have also been moved into a new tools __init__.py file. 

I also added plot_and_publish.py -- simply change the `trigger_name` and add your endpoints to give it a full run. The output should show up on: https://petreldata.net/kanzus/projects/ssx/

### CODE REVIEW

File: nick_testing.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from pprint import pprint
+
+
+class SSXReprocessing(GladierClient):
+    client_id = 'e6c75d97-532a-4c88-b031-8584a319fa3e'
+    gladier_tools = [
+        'tools.publish.SSXPilot',
+    ]
+
+    flow_definition = 'tools.publish.SSXPilot'
```
Comment (ravescovi) [inline nick_testing.py]: This is interesting. I will try to fix mine to look the same later.

File: tools/publish.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
+from gladier.defaults import GladierDefaults
```
Comment (ravescovi) [inline tools/publish.py]: Can you break that into __init__ like the other functions?

File: tools/publish.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                     dry_run=pargs.get('dry_run', False))
+    # These result in lots of output, remove them so they're not in automate
+    del result['new_metadata']
+    del result['previous_metadata']
+    del result['upload']
+
+    return result
+
+
+class SSXPilot(GladierDefaults):
```
Comment (ravescovi) [inline tools/publish.py]: Same here. move to __init__

File: nick_testing.py (hunk 1 above)
Comment (NickolausDS) [reply nick_testing.py]: Ahh, this also requires treating the gladier_kanzus project as a 'package', where 'tools' is the package name. Maybe we should have a different top level folder structured like 'gladier_kanzus/tools/`?

File: tools/publish.py (hunk 2 above)
Comment (NickolausDS) [reply tools/publish.py]: Sure, happy to!

**Review by ravescovi — APPROVED** [review]

**Pull request merged**